src/main/resources/application.properties

```diff
@@ -1,5 +0,0 @@
-spring.social.twitter.appId=DJ1lZnxVadvGsTqiFhPs0jasq
```

Comment:
You might want to regenerate these keys, if you haven't already.

Reply:
was too lazy to do that but found that it is a very easy and short process so it is done. thanks for motivating me :-)